docs/pages/reference/networking.mdx

```diff
@@ -206,3 +215,33 @@ Read more in our [TLS Routing](../architecture/tls-routing.mdx) guide.
 
 </TabItem>
 </Tabs>
+
+### Agent ports
+
+For Teleport processes running agents, e.g., instances of the SSH Service,
+Kubernetes Service, and other services that protect resources in your
+infrastructure, there is no need to open ports on the instance to the public
+internet. 
+
+Teleport agents dial the Teleport Proxy Service to establish a reverse tunnel.
+Client traffic flows via the Proxy Service to the agent, and the agent forwards
+traffic to resources in your infrastructure.
+
+Some Teleport services listen for traffic to one of their proxied resources,
+meaning that you can expose ports on that service's host directly to clients.
+The table below describes the ports that each Teleport Service opens for proxied
+traffic:
+
+| Port | Service | Traffic Type |
+| - | - | - |
+| 3022 | SSH Service | Incoming SSH connections.|
+| 3026 | Kubernetes Service | HTTPS traffic to a Kubernetes API server.| 
+| 3036 | Database Service | Traffic to MySQL databases.|
+| 5432 | Database Service | Traffic to Postgres databases.|
+| 27017 | Database Service | Traffic to MongoDB instances.|
+| 6379 | Database Service | Traffic to Redis instances.|
+| 3028 | Windows Desktop Service | Teleport Desktop Protocol traffic from Teleport clients.|
+
+For applications registered with the Teleport Application Service, the Teleport
```

Comment:
I'm not too certain on this technical accuracy of this statement - might be worth checking with someone else ?

Reply:
This was a guess based on error messages in the source. 

https://github.com/gravitational/teleport/blob/e12a13346333f89374054e603594e948520c9f5a/lib/service/servicecfg/app.go#L110-L119

@r0mant Is it possible to access registered applications directly via the Application Service?

Reply:
Only through the proxy. This paragraph is technically correct, although you can also assign a custom `public_addr` to each application and access it through that, as long as this public address points to proxy and the proxy serves a certificate that's valid for this DNS name.

Reply:
Thanks! I'll remove the DNS details from this paragraph to keep it from getting too long/sidetracked.